tune_sklearn/tune_gridsearch.py

```diff
@@ -51,6 +51,8 @@ class TuneGridSearchCV(TuneBaseSearchCV):
             string or a callable to evaluate the predictions on the test set.
             See https://scikit-learn.org/stable/modules/model_evaluation.html
             #scoring-parameter for all options.
+            For evaluating multiple metrics, either give a list of (unique)
```

Comment:
list or tuple?

Reply:
I believe they all work but yeah need to fix up this documentation. It's from sklearn though

Reply:
Ah ok nice